### DISCUSSION TITLE
[3.0.12] Need a hand for Astroid 3.0.12 Testing

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#613] [Announcements]
Hi All,

Astroid 3.0.12 will be released in the next few days! You can review this version below:

- Test new Editor - CKEditor
- Try use Menu Assignment of Widgets
- New UI/UX of Select Element popup
- Enable/Disable Widget elements in Astroid System plugin

### How to test

-  Clone your site to a test server (Don't do it on live site)
-  Download/Install Astroid 3 dev https://github.com/templaza/astroid-framework/archive/refs/heads/v3.zip
-  Test all function from backend to frontend
-  Please tell me any issue you got.

Thank you all! You are great! We will make Astroid great again, together.

Best Regards,
Sonny

**Comment by Dnsftl** [comment]
Hi Sonny,

This is what I'm encountering now in Astroid 3 dev.

Test Editor - CKEditor

- Image not uploading
- Link not working, text is highlighted address bar opens
- Insert Media, this media (.mp4) is not supported

Enable/Disable Widget elements in Astroid System plugin

Enable/Disable Widget elements in Astroid System plugin, not present in the Astroid plugin

Tested in:

Joomla 5.0.3
Php version 8.2
Template Two
Astroid dev v3

Thanks for all the work you do.

Best Regards,

Dennis

**Comment by rinka88** [comment]
I confirm.
Same characteristics.
php 8.1
MariaDb 10.11

**Comment by deltapapa01** [comment]
Hi Sonny,
i test a little bit with v3 branch (3.0.12), Joomla 5.1.0 Beta 2 and PHP 8.3.3:

Enable/Disable Widget elements in Astroid System plugin -> I can't see a button to enable / disable in System Plugin
Under Widget , Slideshow, Overlay Options, when i set a color, the picture in the slideshow is gone. Every thing i do in the settings, the overlay is located above the entire slideshow:

![image](https://github.com/templaza/astroid-framework/assets/19833064/b035f41c-490f-4868-8be2-3fe25328de58)

![image](https://github.com/templaza/astroid-framework/assets/19833064/8ef4f7ca-ccea-4c40-8b61-0098c71528d9)

Thank you very much for all your work for Astroid and Joomla!!!!!

**Comment by deltapapa01** [comment]
One more quick question: The CK editor is optional, or is it integrated in the Astroid? Because I can't find it :-)

**Comment by sonvnn** [comment]
@Dnsftl @deltapapa01 @rinka88 I re-added enable/disable widget in https://github.com/templaza/astroid-framework/commit/3da94aade0f9565bf427d21463d90f9359b4c9fa

Please help me download v3-branch and test it again.

Thanks & Best Regards,
Sonny

**Reply by Dnsftl** [reply]
@sonvnn

Sorry for the late reply, we had an unexpected death in the family.

Test Editor - CKEditor

- Add Image works, (Only missing a menu where you can choose an image on your computer. I have no idea whether this is the intention or whether it will come in the future)
- Add media in the backend works, fronted only a link is visible, not the video. (Only missing a menu where you can choose an image on your computer. I have no idea whether this is the intention or whether it will come in the future)
- Add link to text works, the link is only visible on the fronted when you hover your mouse over it.
- Enable/Disable Widget elements in Astroid System plugin is present and working properly

Best regard,

Dennis

**Reply by sonvnn** [reply]
> Add Image works, (Only missing a menu where you can choose an image on your computer. I have no idea whether this is the intention or whether it will come in the future)

Yes, Image upload from computer will be release in the future.

> Add media in the backend works, fronted only a link is visible, not the video. (Only missing a menu where you can choose an image on your computer. I have no idea whether this is the intention or whether it will come in the future)

I will improve this feature.

> Add link to text works, the link is only visible on the fronted when you hover your mouse over it.

It depends of your style. Please help me change link color in Template options

**Comment by rinka88** [comment]
Yes, the button to enable and disable widgets has appeared and it works.
Now it's up to the CKE editor

**Comment by Banuk1616** [comment]
I tested V3 Brunch today. But I won't be of much help because I don't have enough knowledge about the Astroid Framework yet.

Assigning widgets to a menu works for me. Enabling and disabling the widgets in System Plugin also seems to work well for me.

I can't find the CKEditor.

In the Grid Widget it seems to me as if the selection of the style under the Read More options partially overwrites or ignores the custom class.

**Reply by rinka88** [reply]
Cke editor is in the text widgets

**Reply by deltapapa01** [reply]
Hi, did you mean this:
![image](https://github.com/templaza/astroid-framework/assets/19833064/5a203f10-400e-4a30-aa6c-b9cec8c3367c)

**Reply by rinka88** [reply]
To do this, you need to download the dev version from the link and install it, which is indicated in the first message.

**Comment by ghost** [comment]
Astroid 3 dev.

Dropdown Alignment under Astroid Menu Option doesn't work for me.

**Reply by sonvnn** [reply]
May you send me your site? Dropdown Alignment still work well on my site.

**Reply by ghost** [reply]
Thank you for your reply. I'm currently trying to get to know all the possibilities of Astroidframe Work and am currently only working on a local development environment.
I don't know why, but now it suddenly works. I must have made a mistake somewhere. Excuse me.
